server/src/main/java/io/crate/metadata/settings/CoordinatorSessionSettings.java

```diff
@@ -42,7 +42,7 @@ public class CoordinatorSessionSettings extends SessionSettings {
 
     private final User authenticatedUser;
     private User sessionUser;
-    private Set<Class<? extends Rule<?>>> excludedOptimizerRules;
+    private Set<Class<? extends Rule>> excludedOptimizerRules;
```

Comment:
Why did you change these declarations? We shouldn't use raw parameterized classes. Actually we should drive the opposite: fix any raw parameterized usages when spotted.

Reply:
See the comment for this.

Reply:
The only option would be a cast to `Class<? extends Rule<?>`.

Reply:
Ok I understood the reason by reading your comment (Sorry slipped through). But as stated, we shouldn't change it into raw parameterized declaration, but rather change the accessing code like suggested in my other comments.